src/utils/csv.ts

```diff
@@ -1,9 +1,8 @@
 import ObjectsToCsv from 'objects-to-csv';
 
-import { log } from './logger';
-
 export const SaveToFile = async (filename: string, data: object[]) => {
+  const parsed = filename.replace(/\s/g, '-');
   const csv = new ObjectsToCsv(data);
-  await csv.toDisk(`./${filename}`);
-  log('file saved:', `./${filename}`);
+  await csv.toDisk(`./${parsed}`);
+  return parsed;
```

Comment:
returns the `parsed` filename. This changes any spaces in the name to `dashes`.

The reason we return it is so we can display an `info` ora message.